dashboard/test/integration/omniauth/google_oauth2_test.rb

```diff
@@ -324,27 +107,28 @@ class GoogleOAuth2Test < ActionDispatch::IntegrationTest
       refute_sign_up_tracking
     end
 
-    test "sign-in from sign-up page" do
-      auth_hash = mock_oauth
+    ### TODO: Create an equivalent test for this in new sign up flow ACQ-2871
```

Comment:
Curious why you commented out here but not in the previous file? It's not a big deal and I don't really have an opinion, so no need to update, just curious.

Also, alternatively, you can skip tests but adding the line `skip` as the first line in the test. Not sure if that's better or worse.

Reply:
My rule of thumb was to fully delete tests I had already recreated for the new flow! So many of these oauth ones already have equivalents I've written, but I didn't remember writing anything like this and wanted to at least make sure we check if we should.

Reply:
ohhh yes fair point. Iirc, I was reading through the existing tests and wondering if the new ones were even necessary or would mirror them at all so just opted to delete and leave the comment.